src/amo/components/HomeHeroBanner/styles.scss

```diff
@@ -61,14 +68,14 @@ $colors: ("blue", #228ae6, #1b6ec2)
 }
 
 // Generate CSS classes depending on the order (indexes) of the Hero sections.
-@for $x from 0 through 3 {
-  @for $y from 0 through 3 {
-    @for $z from 0 through 3 {
+@for $x from 0 through 6 {
+  @for $y from 0 through 7 {
```

Comment:
why 7 here?

Reply:
There are seven items. If I didn't set it to 7 it would out of bounds me on the colour thing... IIRC.

Reply:
It might be fine as 6 though, actually. It being 7 just outputs extra CSS... but if it works with six I'll do a follow-up.

Reply:
what is weird is that it uses 6 on two loops and 7 on that one.

Reply:
You're right, it should be 6.